docs/getting_started/features/pathways.md

```diff
@@ -5,47 +5,212 @@ Pathways contains features that allow to model large transit stations, helping r
+    </p>
+    !!! note ""
+        <p style="font-size:16px">
+        <a href="/documentation/schedule/reference/#stopstxt"><b>stops.txt</b></a> <br>
+        </p>
+
+        | stop_id      | stop_name                                             | location_type | parent_station |
+        |--------------|-------------------------------------------------------|---------------|----------------|
+        | Station_A102 | Main Street station                                   |             1 |                |
+        | A102_B01     | Main Street station - North Platform                  |             0 | Station_A102   |
+        | A102_B02     | Main Street station - South Platform                  |             0 | Station_A102   |
+        | A102_E01     | Main Street station - Entrance/Exit                   |             2 | Station_A102   |
+        | A102_S01     | Main Street station - Top of entrance stairs          |             3 | Station_A102   |
+        | A102_S02     | Main Street station - Bottom of entrance stairs       |             3 | Station_A102   |
+        | A102_S03     | Main Street station - Top of north platform stairs    |             3 | Station_A102   |
+        | A102_S04     | Main Street station - Bottom of north platform stairs |             3 | Station_A102   |
+        | A102_S05     | Main Street station - Top of south platform stairs    |             3 | Station_A102   |
+        | A102_S06     | Main Street station - Bottom of south platform stairs |             3 | Station_A102   |
+        | A102_F01     | Main Street station - Paid side of fare gate          |             3 | Station_A102   |
+        | A102_F02     | Main Street station - Unpaid side of fare gate        |             3 | Station_A102   |
 
 ## Pathway connections
 
 At its foundational level, Pathways offers basic functionality to connect key areas defined in Location types within a station. These connections form pathways, enabling users to obtain precise directions (e. g. from an entrance to the boarding area), which is particularly useful in navigating large and complex transit stations.
 
+**Pre-requirements**: Implement base features and location type feature.
+
 | Files included                   | Fields included   |
 |----------------------------------|-------------------|
-|[pathways.txt](/schedule/reference/#pathwaystxt)|`pathway_id`, `from_stop_id`, `to_stop_id`, `pathway_mode`, `is_bidirectional` |
+|[pathways.txt](/documentation/schedule/reference/#pathwaystxt)|`pathway_id`, `from_stop_id`, `to_stop_id`, `pathway_mode`, `is_bidirectional` |
+
+??? note "Sample Data"
+    <p style="font-size:16px">
+    The following table defines multiple connections (also referred to as pathways) between pre-established nodes, including walkways, stairs and fare gates, while also defining its bidirectionality.
```

Comment:
- we should stick to "locations" or "location types" when talking about all the different locations. Using "node" could create confusion since one single location type is called "generic node"
- we can remove "including" since you call out all pathway modes that appear in the sample. 

```suggestion
    The following sample defines multiple connections (also referred to as pathways) between pre-established locations: walkways (`pathway_mode=1`), stairs (`pathway_mode=2`), and fare gate (`pathway_mode=6`). Bi-directionality is also specified.
```